packages/timeline-state-resolver/src/integrations/sisyfos/$schemas/actions.json

```diff
@@ -6,6 +6,38 @@
 			"name": "Reinitialize",
 			"destructive": false,
 			"timeout": 5000
+		},
+		{
+			"id": "reSyncChannel",
```

Comment:
As far as I can tell this action does not do much from a user perspective. It will pull the external state from Sisyfos but as this state is not used to compare it to the timeline I believe it effectively  will not send any commands?

I think anyone using this action will expect it to do what `setSisyfosChannelState` and we are better off keeping only that. Would love to hear your thoughts on that and maybe I'm just understanding this wrong.